utils/newToken.js

```diff
@@ -0,0 +1,14 @@
+let jwt = require('jsonwebtoken');
+
+const newToken = username => {
+  let expiry = new Date();
+  // expiry.setDate(expiry.getDate() + 7);
+  expiry.setMinutes(expiry.getMinutes() + 1);
```

Comment:
The variable `expiry` is never used?

Reply:
yes, maybe I was sleeping when writing this.